### PR TITLE
Update repo locations

### PR DESCRIPTION
**Pull request by qwertyuioplkjhgfd**
this ~~allows the project to build~~ updates 2 of the outdated dependencies (PlotSquared and FunnyGuilds)
the original repos went offline or something
PreciousStones is still problematic as it no longer compiles (on jitpack.io and in general)

am i doing this right

### CODE REVIEW

File: dough-api/pom.xml [hunk 1]
```diff
@@ -269,9 +273,9 @@
 
         <!-- PreciousStones -->
         <dependency>
-            <groupId>com.github.marcelo-mason</groupId>
+            <groupId>net.sacredlabyrinth.Phaed</groupId>
```
Comment (WalshyDev) [inline dough-api/pom.xml:276]: Would rather not rely on another 3rd party repo. Jitpack is reliable. So why change this?

**Review by WalshyDev — CHANGES_REQUESTED** [review]

File: dough-api/pom.xml (hunk 1 above)
Comment (qwertyuioplkjhgfd) [reply dough-api/pom.xml:276]: Sorry I didn't realize it was in jitpack, when I tried to build it said this was missing so I put in the repository listed in the [repository github](https://github.com/marcelo-mason/PreciousStones/wiki/Developers-API)

**Comment by qwertyuioplkjhgfd** [comment]
it looks like jitpack.io is having problems compiling preciousstones

**Comment by TheBusyBiscuit** [comment]
You are targeting an old PreciousStones version from 2017, it has already been updated in the meantime by others.
After a quick search, https://github.com/elBukkit/PreciousStones seems to be the most up-to-date fork and it compiles successfully on jitpack.

**Review by TheBusyBiscuit — APPROVED** [review]
LGTM
👍🏻 Thanks a lot

**Pull request merged**